invenio_records_resources/records/systemfields/files/manager.py

```diff
@@ -196,7 +192,6 @@ def update(self, key, obj=None, stream=None, data=None, **kwargs):
     @ensure_enabled
     def commit(self, file_key):
         """Commit a file."""
-        # TODO: Add other checks here (e.g. verify checksum, S3 upload)
```

Comment:
Removed this comment because checksum verification/S3 upload belongs at the service-layer and not here at the data-layer.